pkg/packagekit/package_wix.go

```diff
@@ -82,8 +88,31 @@ func PackageWixMSI(ctx context.Context, w io.Writer, po *PackageOptions, include
 	defer wixTool.Cleanup()
 
 	// Use wix to compile into an MSI
-	if err := wixTool.Package(ctx, w); err != nil {
-		return errors.Wrap(err, "running light")
+	msiFile, err := wixTool.Package(ctx)
+	if err != nil {
+		return errors.Wrap(err, "wix packaging")
+	}
+
+	// Sign?
+	if po.WindowsUseSigntool {
+		if err := authenticode.Sign(
```

Comment:
nit, but if we're doing a multi-line method call, it seems a bit convoluted to use the conditional assignment instead of 

```go
err := authenticode.Sign(...)
if err != nil {
// ...
}

Reply:
TBH I like the conditional assignment, solely because I don't have to think about whether `err` was already defined.